package/yast2-devtools.spec

```diff
@@ -54,6 +54,7 @@ Requires:       autoconf
 Requires:       automake
 Requires:       gettext-tools
 Requires:       pkgconfig >= 0.16
+Requires:       rubygem(inifile)
```

Comment:
I'm still not sure about this dependency. The `yast2-devtools` package is also used by plain C++ packages which do not contain any `.desktop` files (like [yast2-core](https://github.com/yast/yast-core) or [yast2-xml](https://github.com/yast/yast-xml)). Adding a full Ruby dependency there would not be nice... :thinking: 

So maybe we should add the dependency only to the packages which actually use the `%yast_metainfo` macro. (Yes, it's still not ideal, but IMHO a bit better solution.)